src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/IWcfDuplexService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ServiceModel;
+
+namespace WcfService
+{
+    [ServiceContract (CallbackContract = typeof(IWcfDuplexServiceCallback))]
+    public interface IWcfDuplexService
```

Comment:
FYI, I opened https://github.com/dotnet/wcf/issues/113 to have a shared file for these interfaces that both the tests and selfhost service can share.  We shouldn't have to maintain 2 copies of every interface.  No need to solve that issue with this PR.